test/e2e/network/netpol/test_helper.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package netpol
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+// prettyPrint a networkPolicy
+func jsonPrettyPrint(policy *networkingv1.NetworkPolicy) string {
+	raw, _ := json.MarshalIndent(policy, "", "\t")
+	return string(raw)
+}
+
+// CreateOrUpdatePolicy fails if it can not create or update the policy in the given namespace
+func CreateOrUpdatePolicy(k8s *Kubernetes, policy *networkingv1.NetworkPolicy, namespace string, isVerbose bool) {
+	if isVerbose {
+		fmt.Println("****************************************************************")
+		framework.Logf("Network Policy creating %s/%s %v", namespace, policy.Name, jsonPrettyPrint(policy))
+		fmt.Println("****************************************************************")
+	}
+
+	_, err := k8s.CreateOrUpdateNetworkPolicy(namespace, policy)
+	framework.ExpectNoError(err, "Unable to create/update netpol %s/%s", namespace, policy.Name)
+}
+
+// CleanPolicies removes network policies
+func CleanPolicies(k8s *Kubernetes, namespaces []string) {
+	err := k8s.CleanNetworkPolicies(namespaces)
+	framework.ExpectNoError(err, "Error occurred while cleaning network policy")
+}
+
+// ValidateOrFail validates connectivity
+func ValidateOrFail(k8s *Kubernetes, model *Model, testCase *TestCase, isVerbose bool) {
```

Comment:
how do you set this argument `isVerbose` ?

Reply:
👍 This is now set from a variable passed to ValidateOrFail calls